packages/amplify-e2e-tests/src/__tests__/rds-refers-to.test.ts

```diff
@@ -96,11 +96,17 @@ describe('RDS Relational Directives', () => {
     });
     port = db.port;
     host = db.endpoint;
-    await addRDSPortInboundRule({
-      region,
-      port: db.port,
-      cidrIp: publicIpCidr,
-    });
+
+    ipAddresses.push(...(await getIpRanges()));
+    await Promise.all(
+      ipAddresses.map((ip) =>
+        addRDSPortInboundRule({
+          region,
+          port: db.port,
+          cidrIp: ip,
+        }),
+      ),
+    );
```

Comment:
nit: since we're doing similar setup for multiple tests, we could consider adding another method to the utils like `addRDSInboundRules(region)` which computes the ipAddresses and adds inbound rule for each of them. Similarly for outbound rules as well. `port` is 3306 in all tests, so it could either be an optional to those methods or skip.

Reply:
We need to store the IP address here so that we can use the same values for remove rule. We can retrieve IP address independent of each function call as it could change in between these 2 invocations.